class One
{
public static void main(String args[])
{

ArithmeticException e=new ArithmeticException();

throw e; //exception is being generated

System.out.println("Back to main");


}

}
